recipes/pydoe2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.1.1" %}
+
+package:
+  name: pydoe2
+  version: {{ version }}
+
+source:
+    fn: pyDOE2-{{ version }}.zip
+    url: https://pypi.io/packages/source/p/pyDOE2/pyDOE2-{{ version }}.zip
+    sha256: c5c0c343bec9b7d0f6650f2d0686802c212ed36ef8b1f895d9cde2ea62768736
+
+build:
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+
+  run:
+    - python
+    - scipy >=0.13
+    - numpy >=1.8
+
+test:
+  imports:
+    - pyDOE2
+
+about:
+  home: https://github.com/clicumu/pyDOE2
+  license_family: BSD
+  license: BSD 3-Clause
+  # https://github.com/clicumu/pyDOE2/pull/7
+  #license_file: LICENSE
```

Comment:
Can we just manually package the `license_file` for now:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and then we can get this merged and update the feedstock when the PR gets merged after the next release.

Reply:
ok